src/components/HookForm/index.jsx

```diff
@@ -508,61 +538,76 @@ export default class HookForm extends Component {
               />
             </FormGroup>
           </ListItem>
+          <ListItem className={classes.displayBlock}>
+            <ListItemText
+              primary={
+                <Typography variant="subtitle1">
+                  Last Hook Fire Attempts
+                </Typography>
+              }
+            />
+            {hookLastFires && (
+              <DataTable
+                items={hookLastFires}
+                headers={['Task ID', 'FiredBy', 'Result', 'Attempted', 'Error']}
+                isPaginate
+                renderRow={hookFire => (
+                  <TableRow key={hookFire.taskId}>
+                    <TableCell>
+                      {(hookFire.result === 'SUCCESS' && (
+                        <Link to={`/tasks/${hookFire.taskId}`}>
+                          <Typography>{hookFire.taskId}</Typography>
+                        </Link>
+                      )) || <Typography>{hookFire.taskId}</Typography>}
+                    </TableCell>
+                    <TableCell>
+                      <Typography>{hookFire.firedBy}</Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Label mini status={hookFire.result.toLowerCase()}>
+                        {hookFire.result}
+                      </Label>
+                    </TableCell>
+                    <TableCell>
+                      <DateDistance from={hookFire.taskCreateTime} />
+                    </TableCell>
+                    <TableCell className={classes.errorTableCell}>
+                      {(hookFire.result === 'ERROR' && (
+                        <ExpansionPanel>
```

Comment:
Instead of having an expansion panel for each error, we should instead have a button that opens a drawer with the error on click. This is similar to https://taskcluster-web.netlify.com/provisioners/aws-provisioner-v1 when clicking the info icon. The drawer can display the `ErrorPanel` component.

![screen shot 2019-02-11 at 10 59 46 am](https://user-images.githubusercontent.com/3766511/52575447-379bcd00-2dec-11e9-887c-0e7fb0d9810c.png)